lib/web_ui/lib/src/engine/pointer_converter.dart

```diff
@@ -22,6 +25,43 @@ class _PointerState {
   double y;
 }
 
+class _GlobalPointerState {
+  _GlobalPointerState() {
+    assert(() {
+      registerHotRestartListener(reset);
+      return true;
+    }());
```

Comment:
Are we landing in the following pattern:

* `reset` for hot restart + tests
* `dispose` for actual code paths that could happen at runtime?

Reply:
The way I think about these is `reset` means the instance is back to its original state and can be used further. OTOH, `dispose` means the object cleaned after itself and can't be used anymore.